tests/test_transforms.py

```diff
@@ -1,94 +1,136 @@
 """Tests for transforms.py module."""
+
 import numpy as np
+import pandas as pd
 import pytest
 
-from distrx.transforms import transform_data, delta_method
-
+from distrx.transforms import (
+    delta_method,
+    transform_data,
+    transform_percentage_change,
+    transform_percentage_change_experiment,
+)
 
 TRANSFORM_DICT = {
-    'log': [
-        np.log,
-        lambda x: 1.0/x
-    ], 'logit': [
-        lambda x: np.log(x/(1.0 - x)),
-        lambda x: 1.0/(x*(1.0 - x))
-    ], 'exp': [
-        np.exp,
-        np.exp
-    ], 'expit': [
-        lambda x: 1.0/(1.0 + np.exp(-x)),
-        lambda x: np.exp(-x)/(1.0 + np.exp(-x))**2
-    ]
+    "log": [np.log, lambda x: 1.0 / x],
+    "logit": [lambda x: np.log(x / (1.0 - x)), lambda x: 1.0 / (x * (1.0 - x))],
+    "exp": [np.exp, np.exp],
+    "expit": [
+        lambda x: 1.0 / (1.0 + np.exp(-x)),
+        lambda x: np.exp(-x) / (1.0 + np.exp(-x)) ** 2,
+    ],
```

Comment:
Might be able to use
https://github.com/ihmeuw-msca/msca/blob/main/src/msca/c2fun/main.py